plotter/sankey_test.go

```diff
@@ -424,14 +424,14 @@ func TestSankey_grouped(t *testing.T) {
 // the input Flows.
 func TestSankey_idempotent(t *testing.T) {
 	flows := []Flow{
-		Flow{
+		{
```

Comment:
These unrelated changes should be here.

Reply:
I've reverted them in the latest commit.